practiceapp/src/main/java/com/practiceapp/practiceapp/entity/PostEntity.java

```diff
@@ -17,6 +20,27 @@
 @Document(collection = "posts")
 public class PostEntity {
 
+
+    @Id
+    @Field
+    private String post_id;
+
+    @Field
+    private String text;
+
+    @Field
+    private Date date;
+
+    @Field
+    private String location;
+
+    @Field
+    private String author_id;
```

Comment:
In order to create a one-to-many relationship between the user and the posts, it should be written as `private UserEntity author`.

Reply:
I updated the relation. You may review again.